Automattic/Gravatar-SDK-Android
# Fix ViewProfileButton paddding when in Empty state

Closes #504

### Description

The `ViewProfile` button in the `ProfileSummary` component had an off padding when in Empty state. It turns out that when in Empty state, the external Modifier was not set on the button, but it was in non-empty state leading to different look.

| Before | After |
|---|---|
| <img width="342" alt="Screenshot 2024-12-19 at 11 50 44" src="https://github.com/user-attachments/assets/bcae32c8-5ae5-44db-8b0b-cfe48a4880f8" /> | <img width="343" alt="Screenshot 2024-12-19 at 11 50 33" src="https://github.com/user-attachments/assets/b63fac68-322c-4d09-90ae-0a5c56507684" /> |

### Testing Steps

1. Use the profile tab in the demo app
2. Display the profile card for a random email that doesn't have an associated Gravatar account 
3. Confirm that the 2nd card in the list looks correct
